packages/badge/Badge.test.tsx

```diff
@@ -1,18 +1,76 @@
-import { Badge } from './Badge';
-
+import { Badge } from './index';
 import { h, mount } from 'bore';
-import { emit } from 'skatejs';
 
 describe( Badge.is, () => {
 
   it( 'should render', () => {
 
-    console.warn('Missing tests for Badge!');
-
     return mount(
       <Badge/>
-    ).wait();
+    ).wait((wrapper) => {
+      expect(wrapper.has('.c-badge')).to.equal(true);
+    });
 
   });
 
+  it( 'should render intrinsic element', () => {
+
+    return mount(
+      <bl-badge></bl-badge>
+    ).wait((wrapper) => {
+      expect(wrapper.has('.c-badge')).to.equal(true);
+    });
+
+  });
+
+  it( 'should render rounded', () => {
+
+    return mount(
+      <Badge rounded>Rounded</Badge>
+    ).wait((wrapper) => {
+      expect(wrapper.has('.c-badge--rounded')).to.equal(true);
+    });
+
+  } );
+
+  it( 'should render ghosted', () => {
+
+    return mount(
+      <Badge ghost>Ghosted</Badge>
+    ).wait((wrapper) => {
+      expect(wrapper.has('.c-badge--ghost')).to.equal(true);
+    });
+
+  } );
+
+  it( 'should render with blaze colors', () => {
+
+    return mount(
+      <div>
+        <Badge color="brand">Brand</Badge>
+        <Badge color="info">Info</Badge>
+        <Badge color="warning">Warning</Badge>
+        <Badge color="success">Success</Badge>
+        <Badge color="error">Error</Badge>
+      </div>
+    ).waitFor((wrapper: any) => {
+      const badges = wrapper.all(Badge);
```

Comment:
hmm maybe we should create helper for this, Smthng like:

```ts
function allElementsShadowReady(...elements){
     return elements.every( elem => elem.node.shadowRoot )
}
```

Usage
```ts
.waitFor((wrapper) => {
  return allShalowElementsShadowReady(wrapper.all(Badge))
})
.then()
```

cc @elmariofredo  @hopkins-tk